sunpy/coordinates/frames.py

```diff
@@ -440,18 +440,35 @@ def represent_as(self, base, s='base', in_frame_units=False):
 @add_common_docstring(**_frame_parameters())
 class Helioprojective(SunPyBaseCoordinateFrame):
     """
-    A coordinate or frame in the Helioprojective Cartesian (HPC) system, which is observer-based.
+    A coordinate or frame in the Helioprojective Cartesian (HPC) system.
 
-    - The origin is the location of the observer.
-    - ``Tx`` (aka "theta_x") is the angle relative to the plane containing the Sun-observer line
-      and the Sun's rotation axis, with positive values in the direction of the Sun's west limb.
-    - ``Ty`` (aka "theta_y") is the angle relative to the Sun's equatorial plane, with positive
-      values in the direction of the Sun's north pole.
-    - ``distance`` is the Sun-observer distance.
+    This is an observer-based spherical coordinate system, with:
+
+    - The origin is the observer location.
+    - The line connecting the poles of the frame is parallel to the component of the Sun's
+      rotation axis that is perpendicular to the observer-Sun line.
```

Comment:
```suggestion
    - The line connecting the poles of the frame is aligned with the projection of the Sun's
      rotation axis onto the plane perpendicular to the observer-Sun line.
```

I found the original confusing, is this clearer?

Reply:
I don't think your suggestion is clearer, but others should chime in